tests/jobs/scala/src/main/scala/RandomWordReceiver.scala

```diff
@@ -0,0 +1,45 @@
+import scala.util.Random
+
+import org.apache.spark.internal.Logging
+import org.apache.spark.storage.StorageLevel
+import org.apache.spark.streaming.receiver.Receiver
+
+class RandomWordReceiver(wordsPerSecond: Float) extends Receiver[String](StorageLevel.MEMORY_ONLY_2) with Logging {
+  val waitFor: Long = (1000L / wordsPerSecond).toLong
+  val inputList: Array[String] = "the quick brown fox jumps over the lazy dog".split(" ")
```

Comment:
Note that this currently has a pretty limited set of words, but this is probably OK for testing.